workflow-templates/im-run-tf-destroy.yml

```
@@ -11,11 +11,15 @@
 # TODO Prerequisites
 #    - Ensure each of the repo-level and env-level secrets used in this workflow have been populated by an admin in your repository.
 
-name: Terraform Destroy
+name: Terraform Destroy 💥
```

Not everyone uses the icons, so I probably would not include that here.
```suggestion
name: Terraform Destroy
```